botCommands/note.js

```diff
@@ -0,0 +1,12 @@
+const { registerBotCommand } = require("../botEngine.js");
+
+const command = {
+  regex: /(?<![\w`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~])\/note(?![\w`!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~)])/,
+  cb: () => { 
+    return `To note, or not to note, that is the question. Please read this: https://docs.google.com/document/d/1fas8xwMRi4GVrnCtp9U5OL7Y9QB6v5H_Wx19m4UQwgI/edit`;
```

Comment:
We really shouldn't be linking to a google doc, as it's very mutable. What happens when someone turns it into an advertisement and all our links from the bot in the past now link to that ad? Or worse.

Reply:
A better solution to this would be to make a markdown file, or link to the discord conversation that someone trustworthy retains ownership of.

Reply:
Okay. For the time being, I guess I will link to the discord conversation by Briggs.

https://discord.com/channels/505093832157691914/505093832157691916/768161823366578176

Reply:
Updated!